docs/detections/prebuilt-rules/rule-details/cyberark-privileged-access-security-error.asciidoc

```diff
@@ -0,0 +1,85 @@
+[[cyberark-privileged-access-security-error]]
+=== CyberArk Privileged Access Security Error
+
+Identifies the occurrence of a CyberArk Privileged Access Security (PAS) error level audit event. The event.code correlates to the CyberArk Vault Audit Action Code.
+
+*Rule type*: query
+
+*Rule indices*:
+
+* filebeat-*
+* logs-cyberarkpas.audit*
+
+*Severity*: high
+
+*Risk score*: 73
+
+*Runs every*: 5 minutes
+
+*Searches indices from*: now-30m ({ref}/common-options.html#date-math[Date Math format], see also <<rule-schedule, `Additional look-back time`>>)
+
+*Maximum alerts per execution*: 100
+
+*References*:
+
+* https://docs.cyberark.com/Product-Doc/OnlineHelp/PAS/Latest/en/Content/PASREF/Vault%20Audit%20Action%20Codes.htm?tocpath=Administration%7CReferences%7C_____3
+
+*Tags*:
+
+* Elastic
+* cyberarkpas
+* SecOps
+* Log Auditing
+* Threat Detection
+* Privilege Escalation
+
+*Version*: 1
+
+*Added ({stack} release)*: 7.14.0
+
+*Rule authors*: Elastic
+
+*Rule license*: Elastic License v2
+
+==== Potential false positives
+
+To tune this rule, add exceptions to exclude any event.code which should not trigger this rule.
+
+==== Investigation guide
+
+*Config*
+
+The CyberArk Privileged Access Security (PAS) Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule.
```

Comment:
```suggestion
The CyberArk PAS Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule.
```

Reply:
We originally had it abbreviated only but decided that spelling it out made more sense since there are many integrations and not all are obvious

Reply:
Totally fine! I try to push for consistent style guidelines when I can, but there are situations (like this one) where clarity should take precedence over best practices.